web/client/components/resources/modals/Save.jsx

```diff
@@ -82,6 +84,8 @@ class SaveModal extends React.Component {
      * @return the modal for unsaved changes
     */
     render() {
+        const canEditResourcePermission = this.props.user.role === 'ADMIN' ||
+        this.props.resource.attributes && this.props.resource.attributes.owner === this.props.user.name;
```

Comment:
```suggestion
        this.props.resource && this.props.resource.attributes && this.props.resource.attributes.owner === this.props.user.name;
```

Reply:
Resource is also initialized in default prop.

Reply:
not sure if resource can be null, but i would do the fix anyway